From afaccb1f73d320ca6f0cfabe661fa0cad3076fd5 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:23:43 -0400
Subject: [redhat] kabi: Adding symbol inc_zone_page_state (mm/vmstat.c)

Message-id: <8314870c8e1e3a0f8b2a3b2afe5d43198b8c2d38.1410510773.git.poros@redhat.com>
Patchwork-id: 96136
O-Subject: [PATCH RHEL7.1 103/233] kabi: Adding symbol inc_zone_page_state (mm/vmstat.c)
Bugzilla: 1071098
RH-Acked-by: Larry Woodman <lwoodman@redhat.com>

Bugzilla: 1071098
    https://bugzilla.redhat.com/show_bug.cgi?id=1071098

    Description:
    ============
    Update zone statics for page, when page writes are in progress.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/inc_zone_page_state
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/inc_zone_page_state
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/inc_zone_page_state

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/inc_zone_page_state b/redhat/kabi/kabi-module/kabi_ppc64/inc_zone_page_state
new file mode 100644
index 0000000..c4d8859
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/inc_zone_page_state
@@ -0,0 +1,2 @@
+#1-
+0x077d552b inc_zone_page_state vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/inc_zone_page_state b/redhat/kabi/kabi-module/kabi_s390x/inc_zone_page_state
new file mode 100644
index 0000000..736d458
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/inc_zone_page_state
@@ -0,0 +1,2 @@
+#1-
+0xdcaf8bdf inc_zone_page_state vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/inc_zone_page_state b/redhat/kabi/kabi-module/kabi_x86_64/inc_zone_page_state
new file mode 100644
index 0000000..8f660ff
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/inc_zone_page_state
@@ -0,0 +1,2 @@
+#1-
+0xd6f95d96 inc_zone_page_state vmlinux EXPORT_SYMBOL
-- 
1.7.1